src/util/config.js

```diff
@@ -27,5 +27,6 @@ export default {
   sentry: env('SENTRY_DSN'),
   gtmId: env('GTM_ID'),
   rpcProxy: env('RPC_PROXY'),
+  syncThreshhold: env('SYNC_THRESHHOLD') || 20,
```

Comment:
new envs should be listed in .env.template and described in the readme. also see about removing checkSyncInterval if we are replacing that logic

Reply:
even better is if we can just use the current SYNC_INTERVAL env, will make updating the helm chart much easier

Reply:
got it, I will do that and add a bit of comment around the SYNC_INTERVAL inside the readme